CONTRIBUTING.rst

```diff
@@ -304,6 +305,24 @@ Debugging Tips
      $ npm install
      $ node_modules/.bin/grunt build # -> dest/*.global.js
 
+
+Branch Model
+------------
+
+Sphinx project uses following branches for developing.
+
+``master``
+    Used for main development.  All improvement and refactoring, bug fixes
+    are allowed.
+
+``X.Y`` (cf. 1.7)
```

Comment:
This one still looks off. What is `cf.`? Could you drop the brackets here, please

Reply:
done